app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,69 @@
+class OrdersController < ApplicationController
```

Comment:
Generally, there isn't a reason to have all of these instances of `@customer` in the orders controller. As far as I can tell from the schema, an order will always have a customer (customer_id, null: false), and thus Active Record can take care of this for you.